cclib/parser/nwchemparser.py

```diff
@@ -1261,6 +1282,27 @@ def extract(self, inputfile, line):
                 self.append_attribute("vibirs", float(line.split()[5]))
                 line = next(inputfile)  # next line
             self.set_attribute("vibfreqs", vibfreqs)
+        # properties related to symmetry
+        if line.strip().find('symmetry #') != -1:
+            symmno = int(line.strip().split()[-1][0:-1])
+            self.set_attribute('symmno', symmno)
+
+        # Grab point group
+        if line.strip().find('symmetry detected') != -1:
+            point_group = line.strip().split()[0]
+            self.set_attribute('point_group', point_group)
+
+        # Grab rotational constants (convert cm-1 to GHz)
+        if line.strip().startswith('A='):

```

Comment:
I don't think this (or `B=`, `C=`) matches anything.  I'm not finding rotational constants in NWChem at all.

Reply:
This is present in `cclib/data/NWChem/basicNWChem7.0/dvb_ir.out`. This certainly means I'll have to add a test for it, so i've added it to the to do list.

Reply:
Not sure where I was looking then...